src/Core/Grid/Factory/FeatureValueGridFactory.php

```diff
@@ -104,15 +111,38 @@ public function getGrid(SearchCriteriaInterface $searchCriteria): GridInterface
         );
     }
 
-    private function modifyDefinition(GridDefinition $definition, FeatureValueFilters $featureValueFilters): void
+    /**
+     * Some modifications are needed in order to fill some required dynamic values coming from request (which are in filters like the $featureId)
+     *
+     * @param GridDefinition $definition
+     * @param FeatureValueFilters $featureValueFilters
+     *
+     * @return void
+     */
+    protected function modifyDefinition(GridDefinition $definition, FeatureValueFilters $featureValueFilters): void
     {
         $definition->setName($this->featureRepository->getFeatureName(
             new FeatureId($featureValueFilters->getFeatureId()),
             new LanguageId($featureValueFilters->getLanguageId())
         ));
 
+        $this->addFilters($definition, $featureValueFilters);
+        $this->addGridActions($definition, $featureValueFilters);
+        $this->addBulkActions($definition, $featureValueFilters);
+    }
+
+    /**
+     * Add filter rows which requires dynamic values from request such as $featureId.
+     *
+     * @param GridDefinition $definition
+     * @param FeatureValueFilters $featureValueFilters
+     *
+     * @return void
+     */
+    protected function addFilters(GridDefinition $definition, FeatureValueFilters $featureValueFilters): void
+    {
         $definition->getFilters()->add((new Filter('actions', SearchAndResetType::class))
-            // action options are added in FeatureValueGridFactory, because they are dynamic and depend on filters
+            // these action options are added in FeatureValueGridFactory, because they are dynamic and depend on filters
```

Comment:
I think you can remove one of the duplicated lines under `->setAssociatedColumn('actions')`